examples/puppetmaster/docker-compose.yml

```diff
@@ -69,3 +69,5 @@ services:
     image: conjurinc/cli5
```

Comment:
can we update this to cyberark/conjur-cli:5 too?

Reply:
wherever it's listed

Reply:
sure